forage-android/src/main/java/com/joinforage/forage/android/ecom/ui/element/ForagePaymentSheet.kt

```diff
@@ -56,76 +67,59 @@ class ForagePaymentSheet @JvmOverloads constructor(
 ) : ConstraintLayout(context, attrs, defStyleAttr),
     ForageElement<ForagePaymentSheet.CombinedElementState>,
     DynamicEnvElement {
-    internal val cardholderNameEditText: TextInputEditText
-    internal val cardNumberEditText: TextInputEditText
-    internal val expirationEditText: TextInputEditText
-    internal val securityCodeEditText: TextInputEditText
-    internal val zipCodeEditText: TextInputEditText
-    private val allEditText: Iterable<TextInputEditText>
 
-    private var onChangeEventListener: StatefulElementListener<CombinedElementState>? = null
+    private val allFields: Iterable<ForagePaymentSheetField>
+
+    val cardholderNameField: CardholderNameField
+    val cardNumberField: CardNumberField
+    val expirationField: ExpirationField
+    val securityCodeField: SecurityCodeField
+    val zipCodeField: ZipCodeField
 
     init {
         inflate(context, R.layout.forage_payment_sheet, this)
 
-        cardholderNameEditText = findViewById(R.id.cardholderNameEditText)
-        cardNumberEditText = findViewById(R.id.cardNumberEditText)
-        expirationEditText = findViewById(R.id.expirationEditText)
-        securityCodeEditText = findViewById(R.id.securityCodeEditText)
-        zipCodeEditText = findViewById(R.id.zipCodeEditText)
-
-        allEditText = setOf(
-            cardholderNameEditText,
-            cardNumberEditText,
-            expirationEditText,
-            securityCodeEditText,
-            zipCodeEditText
+        val cardholderNameEditText: TextInputEditText = findViewById(R.id.cardholderNameEditText)
```

Comment:
(talking) `findViewById` is a pretty old school way of doing things, we should probably use one of the newer patterns instead. For this PR as long as we're consistent with the existing codebase it should be fine.